app/styles/components/_googleMap.scss

```diff
@@ -1,6 +1,7 @@
 .googleMap-container {
   flex: $googleMap-flex;
   order: $googleMap-order;
+  max-width: 300px;
```

Comment:
i was having some issues with the google map overflowing into the state index wrapper bit, but they're mostly my lack of understanding of flexbox.